sqlengine/postgres_engine.go

```diff
@@ -317,6 +317,22 @@ func (d *PostgresEngine) JDBCURI(address string, port int64, dbname string, user
 	return fmt.Sprintf("jdbc:postgresql://%s:%d/%s?%s", address, port, dbname, params.Encode())
 }
 
+const createExtensionPattern = `CREATE EXTENSION IF NOT EXISTS "{{.extension}}"`
+
+func (d *PostgresEngine) CreateExtensions(extensions []string) error {
+	for _, extension := range extensions {
+		createExtensionTemplate := template.Must(template.New(extension + "Extension").Parse(createExtensionPattern))
```

Comment:
I felt unsure so tried writing out the code.
```go
func (d *PostgresEngine) CreateExtensions(extensions []string) error {
	for _, extension := range extensions {
		_, err := d.db.Exec("CREATE EXTENSION IF NOT EXISTS $1", extension)
		if err != nil {
			return err
		}
	}
	return nil
}
```
This errors with something implying Postgres isn't recognising the query parameter:
```
  Expected error:
      <*pq.Error | 0xc420493b00>: {
          Severity: "ERROR",
          Code: "42601",
          Message: "syntax error at or near \"'$1'\"",
          Detail: "",
          Hint: "",
          Position: "32",
          InternalPosition: "",
          InternalQuery: "",
          Where: "",
          Schema: "",
          Table: "",
          Column: "",
          DataTypeName: "",
          Constraint: "",
          File: "scan.l",
          Line: "1082",
          Routine: "scanner_yyerror",
      }
      pq: syntax error at or near "'$1'"
  not to have occurred
```

After some time I haven't reached a firm conclusion whether the problem is me or Postgres. Unless you spot the issue, let's stick with the template?